p4-16/spec/P4-16-spec.mdk

```diff
@@ -2135,6 +2135,8 @@ specified, so their "size" in bits is not specified (it is
 target-specific).
 
 It is also possible to specify an `enum` with an underlying representation.
+These are sometimes called serializable `enum`s, because fields of
```

Comment:
"headers are allowed to have fields with such enum types"

"fields of" sounds confusing.